FreeRTOS/Test/CMock/smp/single_priority_no_timeslice_covg/single_priority_no_timeslice_covg_utest.c

```diff
@@ -413,3 +413,123 @@ void test_task_step_tick_xNextTaskUnblockTime_not_equal( void )
     xTickCount = 0U;
     vTaskStepTick((TickType_t)10U);
 }
+
+/**
+ * @brief AWS_IoT-FreeRTOS_SMP_TC-<TBD>
+ * Two tasks with different priortiy are created in this test.
+ * This test will verify that current core will be requested to yield when resuming
+ * a higher priority task from ISR.
+ *
+ * #define configRUN_MULTIPLE_PRIORITIES                    0
+ * #define configUSE_TIME_SLICING                           0
+ * #define configNUMBER_OF_CORES                            (N > 1)
+ *
+ * This test can be run with FreeRTOS configured for any number of cores
+ * greater than 1 .
+ *
+ * Tasks are created prior to starting the scheduler.
+ *
+ * Task (T1)	    Task (T2)
+ * Priority - 1     Priority - 2
+ * State - Ready    State - Ready
+ *
+ * Suspend the task (T2) before the scheduler started.
+ *
+ * After calling vTaskStartScheduler()
+ *
+ * Task (T1)	    Task (T2)
+ * Priority - 1     Priority - 2
+ * State - Running  State - Suspending
+ *
+ * After calling xTaskResumeFromISR() and resume the task from ISR.
+ *
+ * Task (T1)	    Task (T2)
+ * Priority - 1     Priority - 2
+ * State - Ready    State - Running
+ *
+ * The return value of xTaskResumeFromISR indicates yield required for the core calling
+ * this API.
+ */
+void test_xTaskResumeFromISR_resume_higher_priority_suspended_task( void )
```

Comment:
I think the function name has to begin with 'test' to be parsed by CMOCK. What if we had a function name scheme such as 'test_coverage_<function_name>_<Description>'

Reply:
Agree with the test case name.